pages/api/predictions/[id].js

```diff
@@ -1,20 +1,17 @@
+import Replicate from "replicate";
+
+const replicate = new Replicate({
+  auth: process.env.REPLICATE_API_TOKEN,
+});
```

Comment:
Coulda put this in a reusable `lib/replicate.js` and used it in both API files, but it's so short that it seems like avoiding the indirection is worth the duplication.

Reply:
Actually, do we need both `[id].js` anymore? If we pass `wait: true` into the `replicate.predictions.create` call, we won't need to poll for changes in this endpoint.

Reply:
Seems good to have a spinner thingy that shows status of the prediction as it's running...

Reply:
If it's just a spinner, then you should be able to bind that to a create request (e.g. with an `isLoading`). But I take your point — you're doing the right thing by binding to the prediction as a state object.

So instead of `wait: true`, you could call `replicate.wait(prediction)` after creating it. 

👇 Meet me down here

```

Reply:
I said spinner, but it's actually a thing that displays the value of `status`, e.g. `starting` , `processing`, `completed`

Reply:
Oh sorry, I got the wrong impression of what was going on here from the PR view. Checking it out locally, I now remember how this works. Carry on! 🙃